src/Entity/ShipmentServiceOptions.php

```diff
@@ -91,9 +97,9 @@ public function __construct($response = null)
             }
             if (isset($response->DirectDeliveryOnlyIndicator)) {
                 $this->DirectDeliveryOnlyIndicator = $response->DirectDeliveryOnlyIndicator;
-            }
-            if (isset($response->DeliverToAddresseeOnlyIndicator)) {
-                $this->DeliverToAddresseeOnlyIndicator = $response->DeliverToAddresseeOnlyIndicator;
```

Comment:
i think i mixed up a little, there is obviously no reason to remove it, in my dev branch i was adding more options